Any ideas why the static checks are failing on a problem that doesn't exist?

```tests/task/task_runner/test_standard_task_runner.py:145:52: F821 undefined name 'getpass'```

--- a/airflow/providers/microsoft/winrm/hooks/winrm.py
+++ b/airflow/providers/microsoft/winrm/hooks/winrm.py
@@ -17,13 +17,13 @@
 # under the License.
 #
 """Hook for winrm remote execution."""
-import getpass
 from typing import Optional
 
 from winrm.protocol import Protocol
 
 from airflow.exceptions import AirflowException
 from airflow.hooks.base import BaseHook
+from airflow.utils.platform import getuser
 
 
 # TODO: Fixme please - I have too complex implementation
@@ -203,7 +203,7 @@ class WinRMHook(BaseHook):
                 self.remote_host,
                 self.ssh_conn_id,
             )
-            self.username = getpass.getuser()
+            self.username = getuser()
 
         # If endpoint is not set, then build a standard wsman endpoint from host and port.
         if not self.endpoint:

--- a/airflow/providers/ssh/hooks/ssh.py
+++ b/airflow/providers/ssh/hooks/ssh.py
@@ -16,7 +16,6 @@
 # specific language governing permissions and limitations
 # under the License.
 """Hook for SSH connections."""
-import getpass
 import os
 import warnings
 from base64 import decodebytes
@@ -29,6 +28,7 @@ from sshtunnel import SSHTunnelForwarder
 
 from airflow.exceptions import AirflowException
 from airflow.hooks.base import BaseHook
+from airflow.utils.platform import getuser
 
 
 class SSHHook(BaseHook):  # pylint: disable=too-many-instance-attributes
@@ -175,7 +175,7 @@ class SSHHook(BaseHook):  # pylint: disable=too-many-instance-attributes
                 self.remote_host,
                 self.ssh_conn_id,
             )
-            self.username = getpass.getuser()
+            self.username = getuser()
 
         user_ssh_config_filename = os.path.expanduser('~/.ssh/config')
         if os.path.isfile(user_ssh_config_filename):

--- a/airflow/utils/platform.py
+++ b/airflow/utils/platform.py
@@ -68,4 +68,11 @@ def getuser() -> str:
     try:
         return getpass.getuser()
     except KeyError:
-        return str(os.getuid())
+        # Inner import to avoid circular import
+        from airflow.exceptions import AirflowConfigException
+
+        raise AirflowConfigException(
+            "The user that Airflow is running as has no username; you must run"
+            "Airflow as a full user, with a username and home directory, "
+            "in order for it to function properly."
+        )

--- a/tests/jobs/test_base_job.py
+++ b/tests/jobs/test_base_job.py
@@ -121,7 +121,7 @@ class TestBaseJob:
     @conf_vars({('scheduler', 'max_tis_per_query'): '100'})
     @patch('airflow.jobs.base_job.ExecutorLoader.get_default_executor')
     @patch('airflow.jobs.base_job.get_hostname')
-    @patch('airflow.utils.platform.getuser')
+    @patch('airflow.jobs.base_job.getuser')
     def test_essential_attr(self, mock_getuser, mock_hostname, mock_default_executor):
         mock_sequential_executor = SequentialExecutor()
         mock_hostname.return_value = "test_hostname"

--- a/tests/providers/microsoft/winrm/hooks/test_winrm.py
+++ b/tests/providers/microsoft/winrm/hooks/test_winrm.py
@@ -102,7 +102,7 @@ class TestWinRMHook(unittest.TestCase):
             send_cbt=str(connection.extra_dejson['send_cbt']).lower() == 'true',
         )
 
-    @patch('airflow.providers.microsoft.winrm.hooks.winrm.getpass.getuser', return_value='user')
+    @patch('airflow.utils.platform.getuser', return_value='user')
     @patch('airflow.providers.microsoft.winrm.hooks.winrm.Protocol')
     def test_get_conn_no_username(self, mock_protocol, mock_getuser):
         winrm_hook = WinRMHook(remote_host='host', password='password')

--- a/tests/providers/microsoft/winrm/hooks/test_winrm.py
+++ b/tests/providers/microsoft/winrm/hooks/test_winrm.py
@@ -102,7 +102,7 @@ class TestWinRMHook(unittest.TestCase):
             send_cbt=str(connection.extra_dejson['send_cbt']).lower() == 'true',
         )
 
-    @patch('airflow.utils.platform.getuser', return_value='user')
+    @patch('airflow.providers.microsoft.winrm.hooks.winrm.getuser', return_value='user')
     @patch('airflow.providers.microsoft.winrm.hooks.winrm.Protocol')
     def test_get_conn_no_username(self, mock_protocol, mock_getuser):
         winrm_hook = WinRMHook(remote_host='host', password='password')

